### PR TITLE
Added Datastore to project

### PR DESCRIPTION
**Pull request by MauroGuerrero20**
Datastore is utilized to store comments.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -28,18 +35,46 @@
...
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    List<String> commentsList = new ArrayList();
+
+    Query commentsQuery = new Query("Comment");
+    PreparedQuery results = datastore.prepare(commentsQuery);
+
+    for (Entity entity : results.asIterable()){
+      commentsList.add((String)entity.getProperty("comment"));
```
Comment (VivekShankar) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: add a space between `(String)` and `entity.getProperty(...)`

**Review by VivekShankar — COMMENTED** [review]
This looks good Mauro - can you merge in the updates you made in your previous PR?

**Comment by MauroGuerrero20** [comment]
> This looks good Mauro - can you merge in the updates you made in your previous PR?

Done

**Review by VivekShankar — APPROVED** [review]
LGTM

**Pull request merged**